taf/repository.py

```diff
@@ -0,0 +1,4 @@
+class BaseRepository:
```

Comment:
Should we move this to `repository_tool.py` module?

Reply:
I thought about where to put this class a lot. Still not sure. The idea behind `repository_tool` is to link our TAF Repository with TUF repository. So that does not seem like the best place. I initially thought that there would be more code in this module. Maybe it makes sense to move it to `repository_tool` and move it elsewhere if there is a need to extend it.

Reply:
Or we could remove it completely for now since it does not have any logic.

Reply:
Reoved